tests/0.2.0.json

```diff
@@ -391,7 +381,246 @@
+              },
+              "logical": {
+                "const": "build.recap.Duration"
+              }
+            },
+            "required": ["type", "logical"]
+          },
+          "then": {
+            "properties": {
+              "unit": {
+                "type": "string",
+                "enum": [
+                  "year",
+                  "month",
+                  "day",
+                  "hour",
+                  "minute",
+                  "second",
+                  "millisecond",
+                  "microsecond",
+                  "nanosecond",
+                  "picosecond"
+                ]
+              }
+            },
+            "required": ["unit"]
+          }
+        },
+        {
+          "if": {
+            "properties": {
+              "type": {
+                "const": "bytes"
+              },
+              "logical": {
+                "const": "build.recap.Interval"
+              },
+              "bytes": {
+                "const": 16
+              },
+              "variable": {
+                "const": false
+              }
+            },
+            "required": ["type", "logical", "bytes", "variable"]
+          },
+          "then": {
+            "properties": {
+              "unit": {
+                "type": "string",
+                "enum": [
+                  "year",
+                  "month",
+                  "day",
+                  "hour",
+                  "minute",
+                  "second",
+                  "millisecond",
+                  "microsecond",
+                  "nanosecond",
+                  "picosecond"
+                ]
+              }
+            },
+            "required": ["unit"]
+          }
+        },
+        {
+          "if": {
+            "properties": {
+              "type": {
+                "const": "int"
+              },
+              "logical": {
+                "const": "build.recap.Time"
+              }
+            },
+            "required": ["type", "logical"]
+          },
+          "then": {
+            "properties": {
+              "unit": {
+                "type": "string",
+                "enum": [
+                  "year",
+                  "month",
+                  "day",
+                  "hour",
+                  "minute",
+                  "second",
+                  "millisecond",
+                  "microsecond",
+                  "nanosecond",
+                  "picosecond"
+                ]
+              }
+            },
+            "required": ["unit"]
+          }
+        },
+        {
+          "if": {
+            "properties": {
+              "type": {
+                "const": "int"
+              },
+              "logical": {
+                "const": "build.recap.Timestamp"
+              }
+            },
+            "required": ["type", "logical"]
+          },
+          "$comment": "Not validating timezon field because it's a huge list.",
```

Comment:
```suggestion
          "$comment": "Not validating timezone field because it's a huge list.",
```

Reply:
I spent some time looking for jsonschema definitions of a timestamp field so we could just reference out to that, but I couldn't find one. There is this list... https://github.com/dmfilipenko/timezones.json/blob/master/timezones.json